### PR TITLE
refactor: replace wunderctl init with npx create-wundergraph-app

### PR DESCRIPTION
**Pull request by JivusAyrus**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by StarpTech** [comment]
It should not be merged before releasing the new package first.

**Comment by JivusAyrus** [comment]
> It should not be merged before releasing the new package first.

Yep

**Review by fiam — APPROVED** [review]
LGTM, great work!

File: docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md [hunk 1]
```diff
@@ -4,28 +4,4 @@ pageTitle: WunderGraph - wunderctl init
...
-```
-
-### Publish your API to WunderHub
-
-Use this template to easily publish an API to WunderHub.
-
-```shell
-wunderctl init --template=publish-api
-```
+The cmd `wunderctl init` has been replaced by `create-wundergraph-app`.
```
Comment (StarpTech) [inline docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md:7]: Could you add a link to the docs?

**Review by StarpTech — CHANGES_REQUESTED** [review]

**Comment by fiam** [comment]
> Package was published.

Thanks! I've merged main and  the tests were passing. Assuming CI doesn't get stuck, one more review and this one is good to go.

File: docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md (hunk 1 above)
Comment (JivusAyrus) [reply docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md:7]: create-wundergraph-app is a tag.
So on click it redirects to the docs

Comment (JivusAyrus) [reply docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md:7]: image

Comment (StarpTech) [reply docs-website/src/pages/docs/wunderctl-reference/wunderctl-init.md:7]: :+1:

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**